src/common/SurgeStorage.cpp

```diff
@@ -300,13 +302,43 @@ void SurgeStorage::refreshPatchlistAddDir(bool userDir, string subdir)
       return;
    }
 
-   for (auto& p : fs::directory_iterator(patchpath))
+   /*
+   ** std::filesystem has a recursive_directory_iterator, but between the 
+   ** hand rolled ipmmlementation on mac, expermiental on windows, and 
+   ** ostensibly standard on linux it isn't consistent enough to warrant
+   ** using yet, so build my own recursive directory traversal with a simple
+   ** stack
+   */
```

Comment:
I would for inline comments inside function just use `//`.

```
// std::filesystem has a recursive_directory_iterator, but between the 
// hand rolled ipmmlementation on mac, expermiental on windows, and 
// ostensibly standard on linux it isn't consistent enough to warrant
// using yet, so build my own recursive directory traversal with a simple
// stack
```

Reply:
Nah never use // for multi-line comments.